adapters/telegram/src/polling.ts

```diff
@@ -83,7 +83,7 @@ export namespace HttpPolling {
 
   export const Config: Schema<Config> = Schema.object({
     protocol: Schema.const('polling' as const).required(),
-    pollingTimeout: Schema.natural().role('time').default(Time.minute).description('通过长轮询获取更新时请求的超时 (单位为秒)。'),
+    pollingTimeout: Schema.natural().role('ms').default(25000).description('通过长轮询获取更新时请求的超时 (单位为毫秒)。'),
```

Comment:
```suggestion
    pollingTimeout: Schema.natural().role('ms').default(Time.second * 25).description('通过长轮询获取更新时请求的超时 (单位为毫秒)。'),
```

You could still be able to use a time unit here, which is much clear than counting how many zero-es.